docs/sources/alerting/fundamentals/annotation-label/how-to-use-labels.md

```diff
@@ -19,6 +19,16 @@ This topic explains why labels are a fundamental component of alerting.
 - The Alertmanager uses labels to match alerts for [silences]({{< relref "../../silences/" >}}) and [alert groups]({{< relref "../../alert-groups/" >}}) in [notification policies]({{< relref "../../notifications/" >}}).
 - The alerting UI shows labels for every alert instance generated during evaluation of that rule.
 - Contact points can access labels to dynamically generate notifications that contain information specific to the alert that is resulting in a notification.
-- Labels can be added to an [alerting rule]({{< relref "../../alerting-rules/" >}}). These manually configured labels are able to use template functions and reference other labels. Labels added to an alerting rule take precedence in the event of a collision between labels.
+- Labels can be added to an [alerting rule]({{< relref "../../alerting-rules/" >}}). These manually configured labels are able to use template functions and reference other labels. Labels added to an alerting rule take precedence in the event of a collision between labels (except in the case of [Grafana reserved labels](#grafana-reserved-labels)).
 
 {{< figure src="/static/img/docs/alerting/unified/rule-edit-details-8-0.png" max-width="550px" caption="Alert details" >}}
+
+# Grafana reserved labels
+
+> **Note:** Labels prefixed with `g_` are reserved by Grafana for special use. If a manually configured label is added beginning with `g_` it may be overwritten in case of collision.
+
+The current list of labels created by Grafana and available for use anywhere manually configured labels are:
+
+| Label    | Description                               |
+| -------- | ----------------------------------------- |
+| g_folder | Title of the folder containing the alert. |
```

Comment:
Do you mean to say "The current list of manually configurable labels created by Grafana and available for use are:
?

Reply:
I'm definitely trying to say too much in one sentence. How is this?

```suggestion
Grafana reserved labels can be used in the same way as manually configured labels. The current list of available reserved labels are:

| Label    | Description                               |
| -------- | ----------------------------------------- |
| g_folder | Title of the folder containing the alert. |
```